### PR TITLE
Added blocking clicks back to the enchanting minigames

### PR DESCRIPTION
**Pull request by nopothegamer**
let you use screenshot in et overlay

Fixed rounding issue in pv

Added entrance to "floor one" list + made blur limit at 100 (dungeons map)

Added /neurepomode to toggle item editing and dev mode

Changed "NEUAH is DISABLED! Enable in /neusettings." to /neu

Fixed api key autofill with dg copy chat feature

Added config option to npc retexture

### CODE REVIEW

**Comment by nopothegamer** [comment]
yes it decided to show all the commits idk how github works man

File: src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/EnchantingSolvers.java [hunk 1]
```diff
@@ -309,19 +308,19 @@ public static boolean onStackClick(ItemStack stack, int windowId, int slotId, in
 
                         if (chronomatronReplayIndex < chronomatronOrder.size()) {
                             String chronomatronCurrent = chronomatronOrder.get(chronomatronReplayIndex);
-                            /*if(!NotEnoughUpdates.INSTANCE.config.enchantingSolvers.preventMisclicks ||
+                            if(!NotEnoughUpdates.INSTANCE.config.enchantingSolvers.preventMisclicks ||
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/EnchantingSolvers.java:311]: I thought this was against the rules?

**Comment by IRONM00N** [comment]
also stop restoring your nopo branch and use a new one for every pr, I shouldn't have to see 111 commits

File: src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/EnchantingSolvers.java (hunk 1 above)
Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/EnchantingSolvers.java:311]: check contrib chat

**Comment by nopothegamer** [comment]
> also stop restoring your nopo branch and use a new one for every pr, I shouldn't have to see 111 commits

do I have to name it something different?

File: src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java [hunk 2]
```diff
@@ -176,8 +176,8 @@ public void executeRunnable(int runnableId) {
 
     @Expose
     @Category(
-            name = "Misc Overlays",
-            desc = "Misc Overlays"
+            name = "Todo Overlay",
+            desc = "Todo Overlay"
     )
     public MiscOverlays miscOverlays = new MiscOverlays();
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java:182]: update this too then

File: src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/ItemOverlays.java [hunk 3]
```diff
@@ -310,14 +310,40 @@
     @ConfigAccordionId(id = 5)
     public boolean alwaysShowCrystal = false;
 
+    @ConfigOption(
+            name = "Farming Overlays",
+            desc = ""
+    )
+    @ConfigEditorAccordion(id = 6)
+    public boolean FarmingAccordion = false;
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/ItemOverlays.java:318]: why does this start with a capital?

File: src/main/java/io/github/moulberry/notenoughupdates/ItemPriceInformation.java [hunk 4]
```diff
@@ -77,7 +80,9 @@ public static boolean addToTooltip(List<String> tooltip, String internalname, It
                             }
                             int bazaarSellPrice = (int) bazaarInfo.get("avg_sell").getAsFloat() * stackMultiplier;
                             tooltip.add(EnumChatFormatting.YELLOW.toString() + EnumChatFormatting.BOLD + "Bazaar Sell: " +
-                                    EnumChatFormatting.GOLD + EnumChatFormatting.BOLD + format.format(bazaarSellPrice) + " coins");
+                                    EnumChatFormatting.GOLD + EnumChatFormatting.BOLD +
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/ItemPriceInformation.java:83]: make this a method or something, it shouldn't be repeated this many times

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java [hunk 5]
```diff
@@ -336,6 +337,14 @@ public void processCommand(ICommandSender sender, String[] args) {
         }
     });
 
+    SimpleCommand neuRepoMode = new SimpleCommand("neurepomode", new SimpleCommand.ProcessCommandRunnable() {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:340]: why is this necessary? I feel like this will just confuse people more than it will help

**Review by IRONM00N — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java (hunk 5 above)
Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:340]: Its useful and been asked before

File: src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/ItemOverlays.java (hunk 3 above)
Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/ItemOverlays.java]: Fixed

File: src/main/java/io/github/moulberry/notenoughupdates/ItemPriceInformation.java (hunk 4 above)
Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/ItemPriceInformation.java]: idk man i made it "more clean"

File: src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java (hunk 2 above)
Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java:182]: No because it will reset config

**Comment by IRONM00N** [comment]
@nopothegamer this ready?

**Comment by nopothegamer** [comment]
> @nopothegamer this ready?
> 
> 
Should be yeah

**Review by IRONM00N — APPROVED** [review]
